align/cell_fabric/generators.py

```diff
@@ -8,7 +8,7 @@ def __init__( self, nm, layer, direction, *, clg, spg):
         self.clg = clg
         self.spg = spg
 
-    def segment( self, netName, pinName, center, bIdx, eIdx, *, bS=None, eS=None):
+    def segment( self, netName, pinName, center, bIdx, eIdx, *, bS=None, eS=None, netType="drawing"):
```

Comment:
In this PR, should we also retire the pinName parameter?

Reply:
We need something to indicate that a segment is a pin. We did it originally with "pinName" but we could use a (perhaps optional) boolean field named "isPin" or something like that instead. Is that what you are thinking?

Reply:
Or perhaps, netType now includes "pin" as well. That will work too. There might be a lot of dependences that need to be changed. We'll have to look carefully though the after PNR code that does the checking.

Reply:
I can look into the things that need to be changed to remove pinName and support netType = "pin" instead.